g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh

```diff
@@ -0,0 +1,76 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : azure-storage-blob-go
+# Version               : v0.10.0
+# Source repo           : https://github.com/Azure/azure-storage-blob-go.git
+# Tested on             : UBI 8.5
+# Language              : GO
+# Travis-Check          : False
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Nailusha Potnuru <pnailush@in.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION={1:-v0.10.0}
```

Comment:
there is actually a failure on travis at this line https://app.travis-ci.com/github/ppc64le/build-scripts/builds/244659037, please check and fix

Reply:
done